src/auth/auth.guard.ts

```diff
@@ -7,6 +7,9 @@ import { InvalidTokenException } from './../common/exceptions';
 export class AuthGuard extends NestAuthGuard('jwt') {
   public handleRequest<TUser = any>(err: any, user: any, info: any, context: any, status?: any): TUser {
     if (err || !user) {
+      context
+        .getResponse()
+        .headers({ 'WWW-Authenticate': `Bearer error="invalid_token",error_description="${info.message}"` });
```

Comment:
```suggestion
        .headers({ 'WWW-Authenticate': `Bearer realm=things` });
```

I'd remove the `error` and `error_description` for two reasons:
- The body of the message already contains this information
- When authentication fails we try not to provide any information to avoid exploitations.

Reply:
Thanks, @relu91, that makes asense :)